src/Agent.php

```diff
@@ -624,7 +625,7 @@ public function requestAgent($endpoint): Response
             }
         }
 
-        if (!$response) {
+        if (!$response || $response == null) {
```

Comment:
`!$response` already includes `$response == null`.
```suggestion
        if (!$response) {
```